Ch5&6 ch3 harmonization (#507)
worldbank/dime-data-handbook: chapters/5-processing.tex

  reflecting only that information'',
  depending on the context.
	This applies to data acquired from partners as well as
	original data collected by the research team.
}

It is also possible for a raw dataset to not include an unique identifier,
or that the identifier is not a suitable \textbf{project ID}.\sidenote{
	\textbf{Project ID:} The main ID used in your project to identify
	observations.
	You should never have multiple project IDs for the same unit of observation.
	The project ID must uniquely and fully identify all observations in the project.

> 	See \url{https://dimewiki.worldbank.org/ID\_Variable\_Properties} for more details.}

Reviewer: Why are we repeating the definition here?

Author: As it is the first time it is mentioned in this chapter

Author: Should it maybe be an explanation but not the full definition?

Reviewer: I would think we define only once in book and then can point to wiki in subsequent chapters. Otherwise we will create maintenance problems whenever a definition is updated, needing to do it consistently

Reviewer 2: Agree with Ben, should only define terms once in the book, but linking to a wiki page for later entries makes sense (or mention here that this is discussed in ch3)

Author: ok, now in the defintion is only in ch 3, but in both ch 3 and 5 we link to the new Project ID wiki page. Done in: 885c47eabf04f59cb9b637ae55d64f5ea9a5fdf9

Suitable project IDs should, for example, not involve long strings
that are difficult to work with, such as a person's name,

> or be an ID that is known outside the research team.

Reviewer 2: ```suggestion
or be an ID that is used outside the research team.
```

Author: I think _known_ is the correct word here. Relating to the discussion we just had on Slack, the key is that no-one knows the ID outside the research team. It is an direct extension of the recommendation that only people listed on the IRB should have access to the identified data. If we want to, we can say that. That "the link between study ID and PII should only be known by people listed on the IRB".

And then I think when we publish the data, for example in a replication package, we can include the study ID, and other researchers can use it to combine data in that replication package in new ways. that usage of the study ID outside the research team is fine, as long as there is no way for them to know how it links to PII.

Author: In chapter 3 we already say:

> The project ID must be created by the project team, and the linkage to direct identifiers should only be known to people listed on the IRB.

Author: @mariaruth and @bbdaniels , I updated Maria's original suggestion. Let me know what you think about this.

Reviewer: Yes, I like this, and I would change "person's name" on 121 to just "name", since "person's name" is _obviously_ identifying and should almost never be used at all

Author: > Yes, I like this, and I would change "person's name" on 121 to just "name", since "person's name" is _obviously_ identifying and should almost never be used at all

done in 9ee3119c5ff4ed94f2b383a1ef6201418736cbb3

In such cases, cleaning begins by 
adding a project ID to the raw data.
If a project ID already exists, 
for this unit of observation,

> then you should carefully merge it 

Reviewer 2: the sidenote about terminology (merge vs join) currently on line 166 should move up here, as we use the 'term' merge from here on

Author: done in 93480410479541b119828baa56f6dedc28ca3967

from the master dataset
to the raw data 
using other identifying information.
If a project ID does not exist, 

> then you need to create it,

Reviewer 2: ```suggestion
then you need to generate one,
```

add it to the master dataset, 

> and then merge it to the raw data.

Reviewer 2: ```suggestion
and then merge it back into the raw data.
```
